sklearn/semi_supervised/_label_propagation.py

```diff
@@ -246,7 +246,12 @@ def fit(self, X, y):
         self : object
             Returns the instance itself.
         """
-        X, y = self._validate_data(X, y)
+        X, y = self._validate_data(
+            X,
+            y,
+            accept_sparse=["csc", "csr", "coo", "dok", "bsr", "lil", "dia"],
```

Comment:
I'll prefer to start with "csc" and "csr" and add others if they are requested.

```suggestion
            accept_sparse=["csr", "csc"],
```

These are also the ones that are tested.